python/pyspark/sql/functions.py

```diff
@@ -934,6 +934,25 @@ def lead(col, offset=1, default=None):
     return Column(sc._jvm.functions.lead(_to_java_column(col), offset, default))
 
 
+@since(3.1)
+def nth_value(col, offset, ignoreNulls=False):
+    """
+    Window function: returns the value that is the `offset`\\th row of the window frame
```

Comment:
`\\th`?

Reply:
Yeah, it should have `\\` to distinguish `th` from the previous backquote.